FakeTrello/Models/Contributor.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Web;
+
+namespace FakeTrello.Models
+{
+    public class Contributor
+    {
```

Comment:
Consider what a Contributor is (a user linked to a card)-- could it be represented as navigation properties on your other classes rather than as its own class?